src/query/plan/operator.cpp

```diff
@@ -166,10 +166,11 @@ inline void AbortCheck(ExecutionContext const &context) {
 
 }  // namespace
 
-#define SCOPED_PROFILE_OP(name) ScopedProfile profile{ComputeProfilingKey(this), name, &context};
+#define SCOPED_PROFILE_OP(ref) ScopedProfile profile{ComputeProfilingKey(this), ref, &context};
+#define SCOPED_PROFILE_OP_BY_NAME(name) ScopedProfile profile{ComputeProfilingKey(this), name, &context};
```

Comment:
how are those 2 different, seems to it is the same define, am I right?

Reply:
I guess it does not compile unless 2 defines ?

Reply:
rather have `SCOPED_PROFILE_OP` and `SCOPED_PROFILE_OP_BY_REF` since we want to extend, not to modify

Reply:
Right, #defines can’t be overloaded so you need two of them. Will change the names.